dask/dataframe/tests/test_shuffle.py

```diff
@@ -946,12 +936,12 @@ def test_set_index_on_empty():
         actual = ddf[ddf.y > df.y.max()].set_index("x")
         expected = df[df.y > df.y.max()].set_index("x")
 
-        assert assert_eq(actual, expected, **CHECK_FREQ)
+        assert assert_eq(actual, expected, check_freq=False)
```

Comment:
This is equivalent to what we have in `main` currently. We're just removing the `CHECK_FREQ` variable here since bumping our minimum supported `pandas` version makes the value of `CHECK_FREQ` always a constant 

https://github.com/dask/dask/blob/a0f0838c008467f1d696aaab9f15355faf035f4f/dask/dataframe/tests/test_shuffle.py#L48-L50